newIDE/app/src/UI/Accordion.js

```diff
@@ -27,24 +28,42 @@ type AccordionHeadProps = {|
  */
 export const AccordionHeader = (props: AccordionHeadProps) => {
   return (
-    <MUIAccordionSummary
-      expandIcon={
-        props.expandIcon || (
-          <IconButton size="small">
-            <ExpandMoreIcon />
-          </IconButton>
-        )
-      }
-    >
-      <div style={{ flexGrow: 1, display: 'flex', alignItems: 'center' }}>
-        {props.children}
-      </div>
-      {props.actions && (
-        <div style={{ flexGrow: 0, flexShrink: 0, alignSelf: 'center' }}>
-          {props.actions}
-        </div>
-      )}
-    </MUIAccordionSummary>
+    <Column expand>
+      <Line noMargin expand alignItems="center">
+        <Column noMargin expand>
+          <MUIAccordionSummary
+            style={{
+              flexDirection: 'row-reverse',
+              paddingLeft: 0,
+              paddingRight: 0,
+            }}
+            expandIcon={
+              props.expandIcon || (
+                <IconButton size="small">
+                  <ExpandMoreIcon />
+                </IconButton>
+              )
+            }
+          >
+            <div
+              style={{
+                flexGrow: 1,
+                display: 'flex',
+                alignItems: 'center',
+                marginLeft: 16,
+              }}
+            >
+              {props.children}
+            </div>
```

Comment:
How can I remove the hard coded value (`16`)?
It's to avoid the expand icon to overlap the extension icon.
Should the div be replaced with Line or Colums?

Reply:
We'll allow this hardcoded value I think 👍
And keep the div, it's a special case here and it's hidden in a UI component => all good!